web/config/default.settings.php

```diff
@@ -34,7 +34,7 @@
 $app['csrf.secret'] = 'lkjihgfedcba';
 
 // Log path
-$app['log.path'] = __DIR__.'/../var/log/';
+$app['log.path'] = __DIR__.'/../var/log/' . date('Y-m-d') . '.log';
```

Comment:
This is not a path anymore. Maybe rename it?

Also this would be a  breaking change for existing installations (which is okay, but maybe there is a more elegant way to support this feature and keep existing installations as is).